src/components/page/LittlestarLastTimePodcastSection.tsx

```diff
@@ -65,44 +70,54 @@ const StyledCard = styled.div`
     transition: 0.3s;
     cursor: pointer;
     user-select: none;
-
+    text-align: right;
     svg {
       font-size: 30px;
     }
-
-    &:hover {
+    a:hover {
       opacity: 0.9;
+      color: #fff;
     }
   }
 `
 
-const LittlestarLastTimePodcastSection: React.VFC<{
+const LittlestarLastTimePodcastSection: React.FC<{
   options: {
     title?: string
   }
 }> = ({ options: { title } }) => {
-  const { podcast } = useLatestPodcast()
+  const { currentMemberId } = useAuth()
+  const { lastWatchedPodcastAlbumPodcastProgram } = useLastWatchedPodcastAlbumPodcastProgram(currentMemberId || '')
```

Comment:
can use shorter naming like `lastWatchedPodcastAlbum` or `lastWatchedPodcastProgram`
and the data structure should match the naming
for example
```
const lastWatchedPodcastAlbum = {
   id: ...
   podcastProgram: {}
}
const lastWatchedPodcastProgram = {
   id: ...
   podcastAlbum: {}
}

```